syntax = "proto3";

message PartnerEvent {
    enum Action {
      CREATED = 0;
      UPDATED = 1;
      DELETED = 2;
      ATTRACTION_ADDED = 3;
      LICENSE_ADDED = 4;
    }
    string type_name = 1;
    string name = 2;
    string address = 3;
    string email = 4;
    string phone = 5;
    string id = 6;
    Action action = 7;
}
